cmd/agent/app/options/options.go

```diff
@@ -47,6 +47,9 @@ type Options struct {
 	KubeAPIBurst int
 
 	ClusterCacheSyncTimeout metav1.Duration
+	// ResyncPeriod is the base frequency the informers are resynced.
+	// Defaults to 0.
```

Comment:
Is it possible to add the meaning of the value 0?

Reply:
Sure.